src/main/webapp/map.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Hello Google Maps</title>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCEr8IFDb0oorKv9HmcL-XDPu62EAa8kRY"></script>
+      <script>
```

Comment:
Scripts typically also go in their own JS files that are then included via a <script src="somejsfile.js"></script>